pkg/signing/signing_test.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2020 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package signing
+
+import (
+	"testing"
+
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	fakepipelineclient "github.com/tektoncd/pipeline/pkg/client/injection/client/fake"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	rtesting "knative.dev/pkg/reconciler/testing"
+)
+
+func TestMarkSigned(t *testing.T) {
+	ctx, _ := rtesting.SetupFakeContext(t)
+	// Create a TR for testing
+	c := fakepipelineclient.Get(ctx)
+	tr := &v1beta1.TaskRun{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "my-taskrun",
+		},
+		Spec: v1beta1.TaskRunSpec{
+			TaskRef: &v1beta1.TaskRef{
+				Name: "foo",
+			},
+		},
+	}
+	if _, err := c.TektonV1beta1().TaskRuns(tr.Namespace).Create(tr); err != nil {
+		t.Fatal(err)
+	}
+
+	// Now mark it as signed.
+	if err := MarkSigned(tr, c); err != nil {
+		t.Errorf("MarkSigned() error = %v", err)
+	}
+
+	// Now check the signature.
+	signed, err := c.TektonV1beta1().TaskRuns(tr.Namespace).Get(tr.Name, metav1.GetOptions{})
+	if err != nil {
+		t.Errorf("Get() error = %v", err)
+	}
+	if _, ok := signed.Annotations[ChainsAnnotation]; !ok {
+		t.Error("Taskrun not signed.")
+	}
+}
+
+func TestIsSigned(t *testing.T) {
+	tests := []struct {
+		name       string
+		annotation string
+		want       bool
+	}{
+		{
+			name:       "signed",
+			want:       true,
+			annotation: "true",
+		},
+		{
+			name:       "signed with other string",
+			want:       true,
+			annotation: "baz",
```

Comment:
Should `IsSigned` return true for this case?

Reply:
Changed with the above change.